swal-forms.js

```diff
@@ -46,10 +46,13 @@
     addWayToGetFormValuesInDoneFunction: function(swalArgs) {
       var swalFormInstance = this
       var doneFunction = swalArgs[1]
-      swalArgs[1] = function() {
+      swalArgs[1] = function(isConfirm) {
```

Comment:
the diferent behavior came from Pressing Esc key to cancel the sweetAlert, instead clicking in OK Button.